inspirehep/modules/search/walkers/elasticsearch.py

```diff
@@ -118,11 +118,16 @@ def query(keyword):
 
             fields = self.get_fields_for_keyword(keyword, mode='a')
             if fields == ['authors.full_name', 'authors.alternative_name']:
```

Comment:
I am wondering here, is it more robust to check these 2 fields, or better to check the `author` key (from https://github.com/inspirehep/inspire-next/blob/master/inspirehep/config.py#L167).

In that way we can change where the `author` keyword will look but always keep the exact match behaviour.